sw/device/silicon_creator/lib/chip_info.h

```diff
@@ -0,0 +1,26 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_CHIP_INFO_H_
+#define OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_CHIP_INFO_H_
+
+#include <assert.h>
+#include <stdint.h>
+
+typedef struct chip_info {
```

Comment:
Maybe we should ask the compiler to pack this struct.

Reply:
Neat, packing saves 8 bytes. It's 40 bytes without packing, and 32 with packing.

Reply:
We generally try to avoid `__attribute__((packed))` in favor of hand layout by natural alignment.

See [Embedded Systems: Struct Layouts for Long-term Storage](https://docs.google.com/document/d/12f9aQ5X_M2udJxHw8Q9jkWrg3zLb42ZZXqORajaa2x4/edit) for an explanation and discussion.

Reply:
Thanks for the link. Just removed the `packed` attribute. I'm especially convinced not to pack the struct by the fact that taking the address of a member is UB.